### PR TITLE
add Vietnamese again

### PR DESCRIPTION
**Pull request by vietnam1931**
add Vietnamese

### CODE REVIEW

**Comment by GEOEGII555** [comment]
add vietnam again again

**Comment by elijahr2411** [comment]
Hi, quick question, are you a native Vietnamese speaker or is this machine translation?

**Comment by vietnam1931** [comment]
yes translate Việt Nam/Vietnam

File: static/lang/vn-vn.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
+{
+	"languageName": "Việt Nam",
+	"translatedLanguageName": "Vietnamese",
```
Comment (vietnam1931) [inline static/lang/vn-vn.json:3]: Vietnamese

Comment (vietnam1931) [inline static/lang/vn-vn.json:3]: Vietnamese

**Comment by vietnam1931** [comment]
"translatedLanguageName": "Vietnamese",

**Pull request closed without merging**